src/pages/RegisterAndLogin/Components/Inputs.tsx

```diff
@@ -134,11 +137,7 @@ const Inputs = ({ page }: { page: string }) => {
               children={open ? <VisibleIcon /> : <InvisibleIcon />}
             />
           </InputGroup>
-          {formErrors.password ? (
-            <Text color="red" as="i">
-              {formErrors.password}
-            </Text>
-          ) : null}
+          {passStandard && <PassStandards formData={formData} />}
```

Comment:
create a variable like this `const isRegisterPage= page === "register"` and use is everyhwhere that we used _page === "register"_ .

Here basically you can do it 
`{isRegisterPage&& <PasswordRequirements formData={formData} />}`